datamade/openness-project-nmid
# Update data for spring 2020 and disable races views

## Overview

Add new data for Spring 2020. In addition, remove the "Races" views and links from the site and remove lobbyists from the homepage, per Marjorie's request.

## Testing instructions

- Rerun the import with `python manage.py import_data`
- Redo the search index with `python manage.py make_search_index`
- Run the dev server with `python manage.py runserver`
- Visit http://localhost:8000 and confirm that the "Races" tab no longer appears
- Visit http://localhost:8000 and confirm that only candidates and committees are displayed on the homepage
- Visit http://localhost:8000/races and confirm it returns 404
- Visit http://localhost:8000/candidates/michelle-lujan-grisham-2243/ and confirm you don't see the "Races" section above "Past campaigns"
